notion_diary_RAW_FULL.md

```
@@ -0,0 +1,159 @@
+<page url="https://www.notion.so/1f41669a4c168062aa34ff16f2528814" icon="📝">
+<ancestor-path></ancestor-path>
+<properties>
+{"title":"日記"}
+</properties>
+<content>
+<empty-block/>
+<unknown url="https://www.notion.so/1f41669a4c168062aa34ff16f2528814#1f41669a4c1681e7a46ad0a9e89e1317" alt="button"/>
+# <mention-date start="2026-01-05"/> {toggle="true"}
+	<span color="gray">内容をここに記載…</span>
+---
+# <mention-date start="2026-01-05"/> {toggle="true"}
```

markdownlint-cli2 (0.21.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)